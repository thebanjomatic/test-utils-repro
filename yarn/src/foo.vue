<template>
<div>
  <div>
    <slot name="header"></slot>
  </div>
  <div>
    <slot name="body"></slot>
  </div>
</div>
</template>

<script setup>

</script>