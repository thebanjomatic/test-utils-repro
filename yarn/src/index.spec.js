import {test, expect} from 'vitest';
import {mount} from '@vue/test-utils';
import {h} from 'vue';
import foo from './foo.vue';

test('Can mount', () => {
  var wrapper = mount(foo, {
    slots: {
      body() { return h('div', {}, 'body') },
      header() { return h('div', {}, 'header') },
    }
  })

  expect(wrapper.element).toMatchSnapshot();
})