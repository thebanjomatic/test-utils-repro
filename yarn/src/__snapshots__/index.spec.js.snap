// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Can mount 1`] = `
<div>
  <div>
    
    <div>
      header
    </div>
    
  </div>
  <div>
    
    <div>
      body
    </div>
    
  </div>
</div>
`;
